src/tribol/mesh/CouplingScheme.cpp

```diff
@@ -1464,8 +1466,13 @@ void CouplingScheme::computeCommonPlaneTimeStep( RealT& dt )
                   dt1_check1 = ( dt1_vel_check ) ? exceed_max_gap1 : false;
                   dt2_check1 = ( dt2_vel_check ) ? exceed_max_gap2 : false;
 
+#ifdef TRIBOL_USE_RAJA
+                  RAJA::atomicMax<RAJA::auto_atomic>( &msg[0], static_cast<IndexT>( exceed_max_gap1 ) );
```

Comment:
I think this should be ok right? The entries in `msg` are initialized to the static cast of false, so 0. We are interested in anytime they might be triggered to true, which should be static cast to 1 here, so atomic max makes sense.